catalog_tools/download/download_catalogs.py

```diff
@@ -20,14 +21,14 @@ def apply_edwards(mag_type: str, mag: float) -> pd.Series:
         return pd.Series([mag_type, mag])
 
 
-def download_catalog_sed(
-        start_time: dt.datetime = dt.datetime(1970, 1, 1),
-        end_time: dt.datetime = dt.datetime.now(), min_latitude: float = None,
-        max_latitude: float = None, min_longitude: float = None,
-        max_longitude: float = None, min_magnitude: float = 0.01,
-        delta_m: float = 0.1, only_earthquakes: bool = True,
-        convert_to_Mw: bool = True
-) -> pd.DataFrame:
+def download_catalog_sed(start_time: dt.datetime = dt.datetime(1970, 1, 1),
+                         end_time: dt.datetime = dt.datetime.now(),
+                         min_latitude: float = None,
```

Comment:
If `None` is allowed, then the type needs to reflect that (since `None`is not a `float`)

Python >= 3.10
```
min_latitude: float | None = None,
```

Python >= 3.7
```
from typing import Union
min_latitude: Union[float, None] = None
```
or
```
from typing import Optional
min_latitude: Optional[float] = None
```